autoload/denops/server.vim

```diff
@@ -123,16 +123,28 @@ function! s:on_stderr(data) abort
   echohl None
 endfunction
 
-function! s:on_exit(status, ...) abort dict
+function! denops#server#_on_exit(status) abort
+  call s:on_exit(a:status)
+endfunction
+
+function! s:on_exit(status, ...) abort
+  if s:job is# v:null && s:chan is# v:null
+    return
+  endif
   let s:job = v:null
   let s:chan = v:null
   call denops#util#debug(printf('Server stopped: %s', a:status))
   doautocmd <nomodeline> User DenopsStopped
   if s:stopped_on_purpose || v:dying || v:exiting || s:vim_exiting
     return
   endif
-  " Restart asynchronously to avoid #136
-  call timer_start(g:denops#server#restart_delay, { -> s:restart(a:status) })
+  let addr = get(g:, 'denops_server_addr')
+  if empty(addr)
```

Comment:
```suggestion
  if addr ==# ``
```

Reply:
I have adapted to the way it is written in this repository.
https://github.com/vim-denops/denops.vim/blob/53ce6187e04da044c85b79ef856f1487326a7ac5/autoload/denops/server.vim#L16

Reply:
Hm...